src/pl/plpython/plpy_main.c

```diff
@@ -137,6 +156,18 @@ _PG_init(void)
 	prev_cancel_pending_hook = cancel_pending_hook;
 	cancel_pending_hook = PLy_handle_cancel_interrupt;
 
+#if PY_MAJOR_VERSION >= 3
+	DefineCustomStringVariable("plpython3.python_path",
+							gettext_noop("Python path for plpython3."),
```

Comment:
last comment :
'PYTHONPATH' for plpython3 will be clearer IMO.

Reply:
copy that.